### ISSUE TITLE
Can not login to Pokemon go Server with live maps

### ISSUE THREAD

**Issue by olyx11** [mchristopher/PokemonGo-DesktopMap#359]
I keep getting the following error when trying to run the application. Any suggestions?

main.html:166 2016-07-28 15:48:41,815 [auth_google] [   INFO] Google Login successful.

main.html:166 2016-07-28 15:48:42,874 [     search] [   INFO] Failed to login to Pokemon Go. Trying again.

main.html:166 2016-07-28 15:48:43,874 [auth_google] [   INFO] Google Login successful.

main.html:166 2016-07-28 15:48:44,915 [     search] [   INFO] Failed to login to Pokemon Go. Trying again.

**Comment by AVellucci** [comment]
@olyx11 Please be sure to check
http://ispokemongodownornot.com/   or   https://go.jooas.com/
For Server Status before posting connection issues.
You'll see that PGO logins are currently down 
![pgo-1](https://cloud.githubusercontent.com/assets/11682773/17228443/fd866454-54e0-11e6-9984-b4dfc9f0f74a.PNG)
Please close this issue as it's unrelated to the application.

**Comment by AVellucci** [comment]
@olyx11 Please be sure to check
http://ispokemongodownornot.com/   or   https://go.jooas.com/
For Server Status before posting connection issues.
You'll see that PGO logins are currently down/unstable at this time.
![pgo-2](https://cloud.githubusercontent.com/assets/11682773/17229207/a39cc6aa-54e4-11e6-8637-789bfaf874f6.PNG)
Please close this issue as it's unrelated to the application.

**Comment by olyx11** [comment]
I have run into the same issue for the past two days... The map comes up, but always shows waiting. The only issue I can find is the current issue.

**Comment by olyx11** [comment]
Downloaded the latest version 1.5. I now get the attached error when I launch the application. I have restarted my system, tried running as administrator, verified that I only launched the app one time, and still get the error. Also, still get this even though the servers show up 

main.html:157 server up: null
main.html:145 Error getting location, trying second provider PositionError
main.html:147 Found location [30.4000,-97.7528] by ipinfo.io
main.html:167 2016-07-28 20:18:45,736 [     runserver] [   INFO] Parsed location is: 30.4573/-97.7897/0.0000 (lat/lng/alt)
main.html:167 2016-07-28 20:18:45,742 [        search] [   INFO] Search loop 0 starting
main.html:167 2016-07-28 20:18:45,744 [        search] [   INFO] Attempting login to Pokemon Go.
main.html:167 2016-07-28 20:18:45,749 [   auth_google] [   INFO] Google Login successful.
main.html:172 Python server started!
main.html:172 Got open express port: 8001
main.html:172 Express server started!
main.html:157 server up: http://localhost:8001
main.html:167 2016-07-28 20:18:47,029 [        search] [   INFO] Failed to login to Pokemon Go. Trying again in 5 seconds.
main.html:167 2016-07-28 20:18:52,029 [   auth_google] [   INFO] Google Login successful.
main.html:167 2016-07-28 20:18:53,075 [        search] [   INFO] Failed to login to Pokemon Go. Trying again in 5 seconds.

![livemapserror](https://cloud.githubusercontent.com/assets/20711845/17234682/bc9c4718-5500-11e6-8606-c6b98875de3c.JPG)
